### PR TITLE
Add/menu

### PR DESCRIPTION
**Pull request by aitordiaz**
Add menu for command line interface #3

### CODE REVIEW

File: src/averell/utils.py [hunk 1]
```diff
@@ -284,3 +284,40 @@ def read_features(corpus_folder):
...
     features = sorted(features, key=lambda i: i['poem_title'])
     return features
+
+
+def pretty_string(text, n_words):
+    """Add a line break every number of words into a text to create multiline
+    cells to use in `tabulate_corpora`
+
+    :param text: String to be converted
+    :param n_words: Number of words in which the line break will be added
```
Comment (alvp) [inline src/averell/utils.py:294]: ` Number of words in which the line break will be added`
Esto significa "número de palabras tras las que añadir un salto de línea"? Si es así, quizás mejor "Number of words to add a line break after"?

File: src/averell/utils.py [hunk 2]
```diff
@@ -284,3 +284,40 @@ def read_features(corpus_folder):
...
+    words = text.split()
+    grouped_words = [' '.join(words[i: i + n_words]) for i in
+                     range(0, len(words), n_words)]
+    return '\n'.join(grouped_words)
+
+
+def get_main_corpora_info():
+    """Create dict with the main corpora info saved in CORPORA_SOURCES
+
+    :return: Dictionary with the corpora info to show
```
Comment (alvp) [inline src/averell/utils.py:307]: `Dictionary with the corpora info to show`
I think better `Dictionary with the corpora info to be shown`

File: tests/test_utils.py [hunk 3]
```diff
@@ -156,3 +157,8 @@ def corpora_features():
 def test_read_features(corpora_features):
     output = read_features(Path("tests") / "fixtures")
     assert output == corpora_features
+
+
+def test_pretty_string():
+    output = "This is\na divided\nstring"
```
Comment (alvp) [inline tests/test_utils.py:163]: Maybe I'm being picky but change `divided string` by `split string`?

File: src/averell/utils.py [hunk 4]
```diff
@@ -284,3 +284,40 @@ def read_features(corpus_folder):
...
+    :rtype: dict
+    """
+    table = []
+    for corpus_info in CORPORA_SOURCES:
+        corpus_id = CORPORA_SOURCES.index(corpus_info) + 1
+        table.append({
+            "id": corpus_id,
+            "name": pretty_string(corpus_info["name"], 2),
+            "size": corpus_info["properties"]["size"],
```
Comment (alvp) [inline src/averell/utils.py:316]: If you don't want lines that long maybe you can add `corpus_info["properties"]` as a variable and then just:
`corpus_props = corpus_info["properties"]`
`"size": corpus_props["size"],`

File: src/averell/utils.py (hunk 1 above)
Comment (aitordiaz) [reply src/averell/utils.py:294]: Done

File: src/averell/utils.py (hunk 2 above)
Comment (aitordiaz) [reply src/averell/utils.py:307]: Done

File: tests/test_utils.py (hunk 3 above)
Comment (aitordiaz) [reply tests/test_utils.py:163]: Done

File: src/averell/utils.py (hunk 4 above)
Comment (aitordiaz) [reply src/averell/utils.py]: Done

File: tests/test_averell.py [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 from averell.cli import download
 from averell.cli import export
+from averell.cli import list
```
Comment (versae) [inline tests/test_averell.py:5]: Overwriting the native `list()` function might cause unexpected side effects. Can we change the name of the function while still preserving the command?

Comment (aitordiaz) [reply tests/test_averell.py]: Done

**Pull request merged**